.github/workflows/linux_jdk11.yml

```diff
@@ -1,6 +1,9 @@
 name: Linux JDK 11 GitHub CI
 
-on: [pull_request]
+on:
+  pull_request:
+    paths-ignore:
+      - '**/src/main/resources/GeoServerApplication_*.properties'
```

Comment:
Might cause builds to fail when the French localization is changed, as it's used in at least one test.

Reply:
ahmm I was sure I took care of that...
must have pushed to the wrong repo... let me check

Reply:
indeed pushed to different repo, should be good now